config/locales/en.yml

```diff
@@ -159,17 +159,27 @@ en:
   back: Back
   billing_expense_pdf:
     account: Konto
+    amount: Betrag
+    aoz: AOZ
+    book_text_1: Buchungstext
+    book_text_2: Spensenpauschale
+    chosen_amount: zu überweisender Betrag
```

Comment:
Should be English

Reply:
This concerns a document they will print, and they use no English documents.